Engine/Physics/include/Physics.hpp

```diff
@@ -1 +1,50 @@
+// Copyright 2024 Stone-Engine
+
 #pragma once
+
+#include <glm/glm.hpp>
+
+namespace Stone::Physics {
+
+struct Particle {
+	glm::vec3 position;
+	glm::vec3 velocity;
+	glm::vec3 acceleration;
+	float mass;
+
+	Particle(const glm::vec3& pos, float m)
+		: position(pos), mass(m), velocity(0.0f), acceleration(0.0f) {}
+};
+
```

Comment:
We can move the structures Plane and Sphere to the Scene Module.

Like futures structure Capsule and Box

In a new file Scene/Geometry.hpp